packages/web3-signature/package.json

```diff
@@ -43,7 +43,7 @@
   "dependencies": {
     "@requestnetwork/types": "0.27.0",
     "@requestnetwork/utils": "0.26.0",
-    "web3-eth": "1.3.0"
+    "ethers": "5.0.19"
```

Comment:
ethers v5 is a pretty big change compared to v4 afaik, I would recommend using the same version as the other packages in order to avoid conflicts.

Reply:
Good point :)